spec/clj/tic_tac_toe_clojure/core_spec.clj

```diff
@@ -0,0 +1,98 @@
+(ns tic-tac-toe-clojure.core-spec (:require [speclj.core :refer :all] [tic-tac-toe-clojure.core :refer :all]))
+
+(describe "A board"
+  (it "has 9 cells"
+      (should=
+        ["_" "_" "_" "_" "_" "_" "_" "_" "_"]
+        (create-board)))
+
+  (it "sets a X mark at the correct position"
+      (should=
+        ["X" "_" "_" "_" "_" "_" "_" "_" "_"]
+        (set-position (create-board) 0 "X")))
+
+  (it "sets a X mark O mark at the correct position"
```

Comment:
`sets an X mark and an O mark` :)

Reply:
I would even say `sets an X amrk and then an O mark` 

Also, when devleoping O is kind of sticky as it could be seen as a 0 (zerro). Other example, I and l (i and L)

Reply:
@fabientownsend - Is there a way around that? Do I just need to mention that this is the letter O? Github is kind enough to cross the 0 so they don't look the same, but I'm not sure how else to avoid that.

Reply:
You could call the marks `cross` & `nought`, maybe something like that:
```
(def nought "O")
(def cross "X")
```